### ISSUE TITLE
When I load the Cityscapes using the api there is a error

### ISSUE THREAD

**Issue by wxr5899** [voxel51/fiftyone#1146]
### System information

-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOS Big Sur 11.5.1
-   **FiftyOne installed from (pip or source)**: pip
-   **FiftyOne version (run `fiftyone --version`)**: fiftyone v0.11.2, Voxel51, Inc.
-   **Python version**: 3.7.10

### Commands to reproduce

My python script as below:

```
import fiftyone as fo
import fiftyone.zoo as foz

# The path to the source files that you manually downloaded
source_dir = ".../Developer/Dg/OD/Datasets/Cityscapes"

dataset = foz.load_zoo_dataset(
    "cityscapes",
    source_dir=source_dir,
)

session = fo.launch_app(dataset)
session.wait()
```

### Describe the problem
When I run the script to load the citycrapes dataset it goes wrong and the command output as below

Preparing split 'train' in '.../fiftyone/cityscapes/train'
Parsing fine annotations for split 'train'...
 100% |█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2975/2975 [26.0s elapsed, 0s remaining, 133.4 samples/s]      
Finalizing split 'train'...
Uncaught exception
Traceback (most recent call last):
  File "ReadCityscapes.py", line 9, in <module>
    source_dir=source_dir,
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/zoo/datasets/__init__.py", line 201, in load_zoo_dataset
    **download_kwargs,
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/zoo/datasets/__init__.py", line 121, in download_zoo_dataset
    cleanup=cleanup,
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/zoo/datasets/__init__.py", line 1022, in download_and_prepare
    ) = self._download_and_prepare(split_dir, scratch_dir, split)
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/zoo/datasets/base.py", line 412, in _download_and_prepare
    person_annos=self.person_annos,
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/utils/cityscapes.py", line 112, in parse_cityscapes_dataset
    person_annos_dir,
  File "/Applications/anaconda3/envs/fiftyone/lib/python3.7/site-packages/fiftyone/utils/cityscapes.py", line 269, in _export_split
    exporter = foud.FiftyOneDatasetExporter(split_dir, move_media=False)
TypeError: __init__() got an unexpected keyword argument 'move_media'

### What areas of FiftyOne does this bug affect?

-   [√ ] `Core`: Core `fiftyone` Python library issue

### Willingness to contribute

The FiftyOne Community encourages bug fix contributions. Would you or another
member of your organization be willing to contribute a fix for this bug to the
FiftyOne codebase?

-   [√] No. I cannot contribute a bug fix at this time.

**Comment by brimoor** [comment]
Hi @wxr5899 thanks for reporting this. This patch should fix it https://github.com/voxel51/fiftyone/pull/1147.

You can use this patch yourself by doing a [source install](https://github.com/voxel51/fiftyone#installing-from-source) like so:

```shell
git clone https://github.com/voxel51/fiftyone
cd fiftyone
git checkout --track origin/iss-1146
bash install.bash
```

**Comment by brimoor** [comment]
Hi @wxr5899  👋 This is fixed in `fiftyone==0.11.2.1` and you can upgrade like so:

```shell
pip install --upgrade fiftyone
```

**Comment by wxr5899** [comment]
Thank you，fiftyone really helps me a lot in my research. I’m willing to report the errors and bugs to help fiftyone become better!



发自我的iPhone


------------------ Original ------------------
From: Brian Moore ***@***.***&gt;
Date: Sun,Aug 1,2021 0:42 AM
To: voxel51/fiftyone ***@***.***&gt;
Cc: Xinran Wang ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [voxel51/fiftyone] When I load the Cityscapes using the api there is a error (#1146)
